runtime/src/runtime/policy.rs

```diff
@@ -305,6 +310,8 @@ pub mod policy_constants {
 
     pub const NEW_SECTORS_PER_PERIOD_MAX: usize = 128 << 10;
 
+    pub const NEW_MINER_DEPOSIT: u64 = 500;
```

Comment:
```suggestion
    pub const NEW_MINER_MINIMAL_DEPOSIT: u64 = 500;
```

从设计来看，我们允许 miner deposit >=500, 并把所有送入合约的 value，deposit 到 locked rewards。

Reply:
本来就是要全部送到合约的balance， 现在发消息的时候value要大于等于500， 所有的都会进合约的value，

Reply:
要全部抵押？， 全部抵押就不能创建矿工充值一起完成了。 而且估摸测试又要坏一大片，现在很多测试都是创建矿工的时候打钱进去的。

Reply:
有道理，我可能看错了。我认为你前面的代码是全部抵押了。
只抵押500就可以了。其他放在 available 里面比较合理。 

拿就不用 MINIMAL 这个词了。改回来。 😄